operators/curator-operator/0.0.1/manifests/curator-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,441 @@
+          resources:
+          - reports/status
+          verbs:
+          - get
+          - patch
+          - update
+        - apiGroups:
+          - authentication.k8s.io
+          resources:
+          - tokenreviews
+          verbs:
+          - create
+        - apiGroups:
+          - authorization.k8s.io
+          resources:
+          - subjectaccessreviews
+          verbs:
+          - create
+        serviceAccountName: curator-operator-controller-manager
+      deployments:
+      - label:
+          control-plane: controller-manager
+        name: curator-operator-controller-manager
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              control-plane: controller-manager
+          strategy: {}
+          template:
+            metadata:
+              annotations:
+                kubectl.kubernetes.io/default-container: manager
+              labels:
+                control-plane: controller-manager
+            spec:
+              containers:
+              - args:
+                - --secure-listen-address=0.0.0.0:8443
+                - --upstream=http://127.0.0.1:8080/
+                - --logtostderr=true
+                - --v=0
+                image: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
+                name: kube-rbac-proxy
+                ports:
+                - containerPort: 8443
+                  name: https
+                  protocol: TCP
+                resources:
+                  limits:
+                    cpu: 500m
+                    memory: 128Mi
+                  requests:
+                    cpu: 5m
+                    memory: 64Mi
+              - args:
+                - --health-probe-bind-address=:8081
+                - --metrics-bind-address=127.0.0.1:8080
+                - --leader-elect
+                - --http-port=:8082
+                command:
+                - /manager
+                env:
+                - name: DATABASE_NAME
+                  valueFrom:
+                    secretKeyRef:
+                      key: DatabaseName
+                      name: db-secret-file
+                - name: DATABASE_USER
+                  valueFrom:
+                    secretKeyRef:
+                      key: DatabaseUser
+                      name: db-secret-file
+                - name: DATABASE_PASSWORD
+                  valueFrom:
+                    secretKeyRef:
+                      key: DatabasePassword
+                      name: db-secret-file
+                - name: DATABASE_HOST_NAME
+                  valueFrom:
+                    secretKeyRef:
+                      key: DatabaseHostName
+                      name: db-secret-file
+                - name: PORT_NUMBER
+                  valueFrom:
+                    secretKeyRef:
+                      key: DatabasePort
+                      name: db-secret-file
+                image: surbhi0129/curator-operator-new:latest
```

Comment:
This image is different than `metadata.annotations.containerImage`. It should be the same string.

Reply:
@skanthed i would also add full image path `<regristry-host>/surbhi0129/curator-operator-new:latest`

Reply:
@skanthed i would also recommend to have different tag in image and not use `latest`. Because when you will update operator you don't want to use `latest` for older versions

Reply:
@skanthed just reatg the image to something different than `latest`, something like version.

Reply:
I have updated it to be the same as `metadata.annotations.containerImage.` but still failing the installation test.